Update README.md with Other Instructions (#17)
anshsrtv/leaderboard-project: README.md

1. Select the individual event called "Pull Requests". Let the webhook be `Active`.
1. Make another similar webhook with payload URL `/issue/` appended to it, and select the event "issues".
1. Also create a new Leaderboard object in the admin panel or Python shell with the, username same as the github username you are making your PRs with. 

## Other Instructions
This repository has been created while keeping some good coding practices and situational advantages, and so the following needs to be kept in mind while working with it.

> We keep track of merged PRs and total PRs made by each user. For scoring purposes we have divided the issues into three categories. These categories are identifies by adding labels to the issues.

Reviewer: ```suggestion
We keep track of merged PRs and total PRs made by each user. For scoring purposes we have divided the issues into three categories. These categories are identified by adding labels to the issues.
```

1. Good first issue - These are simpler issues to get you started.
2. Medium issues - These are mildly challenging and worthwhile.
3. Hard issues - These are definitely meant to challenge you.

> We understand that the contributing is itself of great importance. So we have decide to keep reward of all participants that solve atleast 2 medium issues and 1 good first issue. Those are tracked by `milestone_achieved`.

Reviewer: ```suggestion
We understand that the contribution is itself of great importance. So we have decided to reward all those participants who solve atleast 2 medium issues and 1 good first issue. This is taken care of by the boolean `milestone_achieved`.
```

For every issue closed, we increment points. The following needs to be kept in mind in practice.
1. Every issue is assigned to one user only (This is important for proper points distribution and less confusion in the repository.)

> 2. Every issue is solved only by the user assigned to it. The person making the PR must ensure the s/he has been assigned the issue to make sure your contribution is counted.

Reviewer: ```suggestion
2. Every issue is solved only by the user assigned to it. The person making the PR must ensure that s/he has been assigned the issue to make sure his/her contribution is counted.
```
